teachertool/src/components/CriteriaResultEntry.tsx

```diff
@@ -36,23 +38,29 @@ const AddNotesButton: React.FC<AddNotesButtonProps> = ({ criteriaId, setShowInpu
 
 interface CriteriaResultNotesProps {
     criteriaId: string;
-    notes?: string;
 }
 
-const CriteriaResultNotes: React.FC<CriteriaResultNotesProps> = ({ criteriaId, notes }) => {
+const CriteriaResultNotes: React.FC<CriteriaResultNotesProps> = ({ criteriaId }) => {
     const { state: teacherTool } = useContext(AppStateContext);
+    const [value, setValue] = useState(teacherTool.evalResults[criteriaId]?.notes ?? "");
```

Comment:
I'm having difficulty understanding why `value` state was introduced. Was it something to do with initializing DebouncedTextArea?

Reply:
Hm. I genuinely can't recall. I'll undo it.